packages/attendance/src/components/AttendanceComponent.js

```diff
@@ -778,11 +786,7 @@ const CalendarComponent = ({
   const handleAttendaceData = (attendance, day) => {
     let isToday = moment().format("YYYY-MM-DD") === day.format("YYYY-MM-DD");
     let isAllowDay = false;
-    if (moment().format("HH:MM") <= manifest?.["class_attendance.submit_by"]) {
-      isAllowDay = true;
-    } else if (
-      manifest?.["class_attendance.previous_attendance_edit"] === "true"
-    ) {
+    if (manifest?.["class_attendance.previous_attendance_edit"] === "true") {
```

Comment:
class_attendance.previous_attendance_edit is it coming from config api?

Reply:
Yes